### PR TITLE
feat: add additional color formats

### PR DESCRIPTION
**Pull request by cagespence**
Add `hslString`, `hslArray`, and CSS `keyword` as optional color formats.

All color conversions in this package now use `color-convert`.

https://github.com/jonyw4/color-thief-react/pull/132#discussion_r666249362

### CODE REVIEW

File: src/types/ColorFormats.ts [hunk 1]
```diff
@@ -4,5 +4,15 @@
...
+ * `hslArray` returns a array of hsl. `[0, 0, 0]`
  */
-export type ColorFormats = 'rgbString' | 'hex' | 'rgbArray';
+export type ColorFormats =
+  | 'rgbString'
+  | 'hex'
+  | 'rgbArray'
+  | 'hslString'
+  | 'hslArray'
+  | 'keyword';
```
Comment (jonyw4) [inline src/types/ColorFormats.ts:18]: I think that need to explain the 'keyword' color format.

**Review by jonyw4 — CHANGES_REQUESTED** [review]
Nice work @cagespence! 
I just left a comment to update the jsdoc in a file. 
I think that is good to put into the README file these new features and which lib that we are using.

File: src/types/ColorFormats.ts (hunk 1 above)
Comment (cagespence) [reply src/types/ColorFormats.ts:18]: Updated the Readme.

**Comment by jonyw4** [comment]
Awesome @cagespence! The final thing, recently I update the dependencies from this repo. So I ask you to:
1. Copy the `yarn.lock` from main and commit
2. Update this branch with main
3. Run the command to add the new dependency again
4. Commit and push

This will resolve the conflict and will maintain the updated that i did

**Review by jonyw4 — APPROVED** [review]

**Pull request merged**